MHVLib/Models/Generated/MHVGoal.m

```diff
@@ -98,9 +133,9 @@ + (NSDictionary *)objectParametersMap
  * This method is used by `JSONModel`.
  */
 + (BOOL)propertyIsOptional:(NSString *)propertyName {
-
-  NSArray *optionalProperties = @[@"_id", @"name", @"_description", @"startDate", @"endDate", @"goalType", @"recurrenceMetrics", @"range", @"additionalRanges"];
-  return [optionalProperties containsObject:propertyName];
+    
+    NSArray *optionalProperties = @[@"_id", @"name", @"_description", @"startDate", @"endDate", @"goalType", @"recurrenceMetrics", @"range", @"additionalRanges"];
```

Comment:
I think you're right that these probably aren't needed. Removed from the swagger template.